subiquity/controllers/ceph.py

```diff
@@ -23,6 +23,13 @@
 
 
 class CephDiskController(BaseController):
+    signals = [
```

Comment:
I don't know how the code is used, but would a dictionary be better here... and below?